src/ui/dialogs/omemoDevices.ts

```diff
@@ -96,6 +136,16 @@ class OmemoDeviceDialog {
 
          listElement.append(element);
       }
+
+      let datauri = this.buildQRCode(devices[0].getAddress().getName(), fingerprints);
+      if (listElement.hasClass('jsxc-omemo-owndevices'))
+      {
+          $('.jsxc-own-device-qrcode-own').attr('src',datauri);
```

Comment:
Could conflict with other elements on the website. Would be better to use `listElement.prev('.jsxc-qr-code').find('img')`.

Reply:
ok changed...